README.md

```diff
@@ -69,6 +69,28 @@ By default, the only Middleware enabled is the Command Handler support. You can
 
 Check the [Tactician docs](http://tactician.thephpleague.com/) for more info and a complete list of middleware. 
 
+### Extra Bundled Middleware
+
+This bundles ships with a few pre-configured middlewares, they can be enabled using the method above by just listing their ids.
+
+#### Validator Middleware (tactician.middleware.validator)
+
+The validator middleware will plug into Symfony's Validator (@validator) and will throw and exception if the command is not valid.
+
+Constraints can be added via configuration or annotations like in default Symfony practices, please refer to [their docs](http://symfony.com/doc/current/book/validation.html). 
+
+The middleware will throw an `InvalidCommand` Exception that will contain the command and the `ContraintViolationList` returned by the validator.
+
+#### Locking Middleware (tactician.middleware.locking)
+
+This middleware is bundled in Tactician, please refer to [the official documentation](http://tactician.thephpleague.com/plugins/locking-middleware/) for details.
+
+#### Command Handler Middleware (tactician.middleware.command_handler)
+
+**Always ensure this is the last middleware listed**
+
+While not listed this is the core of Tactician and handles executing commands, it should always be enabled.
```

Comment:
As discussed in #12 having the command handler middleware is not a hard requirement to work with tactician. My suggestion for these texts:
```
**Always ensure this is the last middleware listed if you use it**

This is the most common middleware you probably use with Tactician as it handles executing commands. So it should be enabled in most cases.